README.md

```diff
@@ -117,7 +117,7 @@ PRs to add additional projects welcome!
 
 ### Running with Optuna's Docker images?
 You can use our docker images with the tag ending with `-dev` to run most of the examples.
-For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python examples/pytorch/pytorch_simple.py`.
+For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python pytorch/pytorch_simple.py`.
```

Comment:
How about mounting the working directory to the container? 
The `examples` directory will be removed from `optuna/optuna`, and users need to get the example script somewhere else.

```suggestion
For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm -v $(pwd):/prj -w /prj optuna/optuna:py3.7-dev python pytorch/pytorch_simple.py`.
```

Reply:
You are right, thanks!